Yoast/wordpress-seo
# Change assessment calls in taxonomy assessor

## Summary

This PR can be summarized in the following changelog entry:

* n/a

## Relevant technical choices:

* Changes the calls and imports in the taxonomy assessor so that they are up-to-date with the refactored code in YoastSEO.js. Also adds specs to the taxonomy assessor.
* Bumps the YoastSEO.js version to v1.36.0

## Test instructions

This PR can be tested by following these steps:

Add a new category page and make sure that the following assessments can be triggered and that no console errors appear:
These assessments should appear right at the beginning:
* MetaDescriptionLength
* TaxonomyTextLength
* TitleWidth
* TitleKeyword
* UrlKeyword

Add a 150 word text; this will also trigger:
* IntroductionKeyword
* KeywordDensity

Fill in a long slug, e.g.  `long long long long long long long long long long long long long long`; this should trigger: 
* UrlLength

Add a stop word to your slug, e.g. `the`; this should trigger:
* urlStopWords

Make your keyword very long, e.g. `long long long long long long long long long long long long long long`; this whould trigger:
* KeyphraseLength

Add a stopword to your keyword, e.g. `the`; this should trigger:
* KeywordStopWords

Add a meta description (and make sure you have a keyword set); this should trigger:
* MetaDescriptionKeyword

## Quality assurance


Fixes #10400
